collab-service/src/message_handler/internal/internal_message_types.d.ts

```diff
@@ -0,0 +1,23 @@
+const enum ConnectionFlag {
```

Comment:
Please help to rename to opcode thanks.
```suggestion
const enum ConnectionOpCode {
```